README.md

```diff
@@ -87,23 +87,23 @@ You can also download the PowerShell binary archives for Windows, macOS and Linu
 [rl-arm64]: https://github.com/PowerShell/PowerShell/releases/download/v7.0.0/powershell-7.0.0-linux-arm64.tar.gz
```

Comment:
@adityapatwardhan, your last commit had 2 failures in `PowerShell-CI-windows`
PSVersionTable.Verify PSCompatibleVersions has no unknown PowerShell entries
```powershell
Expected collection @('1.0', '2.0', '3.0', '4.0', '5.0', '5.1', '6.0', '6.1', '6.2', '7.0') to contain '7.1', but it was not found.
at <ScriptBlock>, D:\a\1\s\test\powershell\Host\PSVersionTable.Tests.ps1: line 174
174:             $version | Should -BeIn $powerShellVersions
```
Requires tests.Version checks.Check for version 7.1
```powershell
Expected an exception, with type [System.Management.Automation.ScriptRequiresException] to be thrown, but no exception was thrown.
at <ScriptBlock>, D:\a\1\s\test\powershell\Language\Scripting\Requires.Tests.ps1: line 90
90:             { . $File } | Should -Throw -ExceptionType ([System.Management.Automation.ScriptRequiresException])
```